docs/reference/snaps.rst

```diff
@@ -0,0 +1,76 @@
+.. _snap_reference:
+
+===============
+Snap Versions
+===============
+
+There are several checkbox snaps and channels to choose from. You can
```

Comment:
In order to avoid confusion between the word checkbox and our software, it's better to uppercase the C: "Checkbox".

(This advice applies in other places in this PR)